app/js/components/drugOrderEntry/ActiveOrders.jsx

```diff
@@ -298,13 +308,17 @@ ActiveOrders.propTypes = {
   activeOrders: PropTypes.shape({}).isRequired,
   showResultCount: PropTypes.string,
   pageCount: PropTypes.number,
+  selectedOrder: PropTypes.object,
+  activity: PropTypes.string,
 };
 
 ActiveOrders.defaultProps = {
   drugOrder: {
     loading: false,
     activeOrders: [],
   },
+  selectedOrder: {},
+  activity: '',
```

Comment:
I don't think the changes in the file are needed since this component(`ActiveOrders`) would be removed as the Active Orders are already displayed on the Orders List page.
If it won't break anything, I suggest you revert the changes.